src/sentry/testutils/region.py

```diff
@@ -15,10 +14,15 @@ def override_regions(regions: Iterable[Region]):
     because the region mapping may already be cached.
     """
 
-    mapping = _RegionMapping(regions)
+    mapping = region._RegionMapping(regions)
 
-    def override() -> _RegionMapping:
+    def override() -> region._RegionMapping:
         return mapping
 
-    with mock.patch("sentry.types.region._load_global_regions", new=override):
+    existing = region._load_global_regions
+    region._load_global_regions = override
```

Comment:
Similarly here. If we want to commit to making these externally visible, even if just for `testutils`, maybe we should bite the bullet and remove the leading underscore.

If we do, we should also do so for `_RegionMapping`, which would also be a good opportunity to rename it per @GabeVillalobos's feedback about "mapping" being ambiguous. Perhaps `GlobalRegionDirectory`.

Reply:
Ok sure.